com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/VaryingMesh.hlsl

```diff
@@ -22,7 +22,9 @@ struct AttributesMesh
 #ifdef ATTRIBUTES_NEED_COLOR
     float4 color        : COLOR;
 #endif
-
+#ifdef DOTS_INSTANCING_ON
+    uint vertexID        : SV_VertexID;
```

Comment:
Note: be sure this compile on DX12 / PS4 / Xbox. We have get various surprise with system sematic ordering on those platform (with double sided, instance id etc...)

also we need to use VERTEXID_SEMANTIC and not SV_VertexID

Reply:
Question: Guess for now Vertex_ID is only used with skinned mesh right?
so it should not be define when DOTS_INSTANCING_ON is on, but when DOTS_SKINNING_INSTANCING_ON ? (which doesnt exist but you get the idea). Just wondering :)

Reply:
ok reading more code it seems that we can only know dynamically if we use skinning or not, so fine.